README.md

```diff
@@ -93,6 +93,23 @@ We use Code generation to generate our types for our GraphQL queries and mutatio
 - Each query and mutation should have a unique name.
 - Since query analysis for type generation occurs statically we cant place dynamic variables with in query strings we instead have to hard code the variable in the query or pass it in as query variable.
 
+## Testing
+Spruce has a combination of unit tests using Jest, and integration tests using Cypress.
+
+### Unit tests
+TODO: write more here. The Jest tests can be run by typing `yarn test`
+
+### Integration tests
+At a high level, we use Cypress to start a virtual browser that is running Spruce. Cypress then is able to run our test specs, which tell it to interact with the browser in certain ways and makes assertions about what happens in the UI. Note that you must be running the Evergreen server on localhost:9090 for the front-end to work.
```

Comment:
I think we should also note that our cypress npm scripts also rely on spruce running on localhost:3000.
Cypress tests can be ran against a chrome browser locally by running `yarn cy:open`